### PR TITLE
Remove trailing slashes on svn checkout

### LINKED ISSUE #4934: Remove trailing slashes on svn checkout
If someone enters an svn repo like `https://svn.code.sf.net/p/docutils/code/`, it would be concatenated with the version `/trunk/`, so we ended up with `https://svn.code.sf.net/p/docutils/code//trunk`

Related code

https://github.com/rtfd/readthedocs.org/blob/7f2b831ac18ee839535254e9b14e595621ebf653/readthedocs/vcs_support/backends/svn.py#L61

### PR DESCRIPTION
**Pull request by dojutsu-user**
Fixes #4934

### CODE REVIEW

**Comment by stsewd** [comment]
I'm not so familiar with svn, but I guess we could do this a little more robust. Could be the case where the branch/tag doesn't have a `/`.

**Comment by dojutsu-user** [comment]
@stsewd 
I have implemented the `lstrip` to deal with case in your comment above.

File: readthedocs/vcs_support/backends/svn.py [hunk 1]
```diff
@@ -58,7 +58,11 @@ def up(self):
     def co(self, identifier=None):
         self.make_clean_working_dir()
         if identifier:
-            url = self.base_url + identifier
+            base = self.base_url.rstrip('/')
+            tag = ''
+            if identifier:
```
Comment (dojutsu-user) [inline readthedocs/vcs_support/backends/svn.py]: Added this if-condition because if `identifier` is `None`, then the `lstrip()` would throw an error.

**Comment by dojutsu-user** [comment]
@stsewd 
okay... that would be great.
One thing, since there are no tests for svn as of now, so where should you thing the tests should be?
I am thinking to create a new test file named - `test_backend_svn.py` ?

**Comment by stsewd** [comment]
yeah, that sounds great

**Comment by stsewd** [comment]
Actually, I wanted to start to separate the test files, some are very big already, not sure anyway.

**Comment by dojutsu-user** [comment]
@stsewd 
Okay

**Pull request merged**